proto-google-cloud-spanner-v1/src/main/java/com/google/spanner/v1/ExecuteSqlRequest.java

```diff
@@ -402,6 +402,9 @@ private QueryMode(int value) {
      * SPANNER_SYS.SUPPORTED_OPTIMIZER_VERSIONS. Executing a SQL statement
      * with an invalid optimizer version will fail with a syntax error
      * (`INVALID_ARGUMENT`) status.
+     * See
```

Comment:
I assume that the changes to this file will be removed later, as this is part of the generated code.

Reply:
Yes, I did add them here, so that the build would pass.